src/Transform/ONNX/Decompose.td

```diff
@@ -49,6 +49,8 @@ def GetNullIntegerAttr : NativeCodeCall<"IntegerAttr()">;
 
 def GetNullStringAttr : NativeCodeCall<"StringAttr()">;
 
+def GetNoneStringAttr : NativeCodeCall<"$_builder.getStringAttr(\"none\")">;
```

Comment:
Different from GetNullStringAtt?

Reply:
Good observation!

I've updated the code to use `getNullStringAttr` instead